helm-charts/inbucket/Chart.yaml

```
@@ -3,7 +3,7 @@ description: Disposable webmail server (similar to Mailinator) with built in SMT
 name: inbucket
 type: application
 appVersion: 3.0.0
-version: 2.2.2
+version: 2.2.3
```

```suggestion
version: 2.3.0
```

this is adding new features but those are not breaking change so a minor is okay